### DISCUSSION TITLE
How can I get back the response from server

### DISCUSSION THREAD

**Discussion by void-m4110c** [rpldy/react-uploady#133] [Q&A] [answered]
Hi,

I'm uploading a csv file and some additional parameters to a server.

It's just an upload button to select the file in an uploady component, which is embedded in a table:
```javascript
            <TableRow>
              <TableCell>{constructLineDropdown()}</TableCell>
              <TableCell>{constructDatePicker('StartDate')}</TableCell>
              <TableCell>{constructDatePicker('EndDate')}</TableCell>
              <Uploady
                clearPendingOnAdd
                multiple={false}
                accept=".csv"
                params={{
                  lineId: capLineId,
                  startDate: startDate.toISOString().split('T')[0],
                  endDate: endDate.toISOString().split('T')[0],
                }}
                destination={{ url: props.uploadURL }}
              >
                <UploadButton text={i18n.translateToString('UploadFile')} />
              </Uploady>
            </TableRow>
```

I know that the upload works correctly, but is there any way that I can get the HTTPResponse from this upload?

Thanks!

**Comment by yoavniran** [comment, marked as answer]
Hey @void-m4110c 
yes, of course you can access the response.

There are two ways to do this. Using [event listeners](https://github.com/rpldy/react-uploady/blob/master/guides/UploaderEvents.md) or [hooks](https://github.com/rpldy/react-uploady/tree/master/packages/ui/uploady#useitemfinishlistener-event-hook) 

the response will be available through the item's `uploadResponse` property.

**Reply by void-m4110c** [reply]
Thank you very much, and sorry again for the misunderstanding

**Reply by void-m4110c** [reply]
Is there a way to get the HTTP-Statuscode (200 | 404 | 500 etc.)  from that `item.uploadResponse`? I can only see headers and data in that response. Or is there any other way?

**Reply by yoavniran** [reply]
thats actually not available currently. interestingly, it wasnt requested until now. I will add it soon.

**Reply by void-m4110c** [reply]
Great! Thank you very much

**Reply by yoavniran** [reply]
@void-m4110c 
https://github.com/rpldy/react-uploady/pull/138 adds `uploadStatus` to batch items. 
Will release soon

**Reply by yoavniran** [reply]
version 0.10.0 just released
uploadStatus (code) is now available
